project/Dependencies.scala

```diff
@@ -6,7 +6,7 @@ object Dependencies {
   val Scala213 = "2.13.6"
   val ScalaVersions = Seq(Scala212, Scala213)
 
-  val AkkaVersion = System.getProperty("override.akka.version", "2.6.9")
+  val AkkaVersion = System.getProperty("override.akka.version", "2.6.15")
```

Comment:
this is because of the AkkaManagement update?

Reply:
No. I was trying to align with the version we use in Akka Platform Guide.

Reply:
The dependency strategy we try to follow for these libraries is to not update the Akka version unless necessary (such as a new feature needed by the library.

Reason is that then users can choose to stay on an older Akka version instead of being enforced to the latest.

Users can use the latest by defining that in their application build.

Akka is only backwards compatible, not forwards compatible. Meaning that users cannot downgrade to a lower version than used by the libraries.

Reply:
Done. This PR is smaller now. :-)